avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -582,6 +589,7 @@ def updatePositionC(cfg, particles, dem, force, DT):
   outOfDEM = np.array(dem['outOfDEM'], dtype=bool)
   # read particles and fields
   cdef double[:] mass = particles['m']
+  cdef double[:] idRel = particles['idRel']
```

Comment:
idRel stands for release? should we not call it boundary or something like that? then we could use it also if ever we need boundary particles for a dam or so

Reply:
I renamed them to idFixed- so to make it clear these do not move - if we need it for another case - we can use the functions to create them and give it a meaningful name - as for now they are removed after the iniStep